<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/pozadina/slikeCalendar/14.jpg);</string>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4"/>
    </item>
    <item row="1" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/pozadina/slikeCalendar/14.jpg);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="calendarPage">
       <property name="styleSheet">
        <string notr="true">background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCalendarWidget" name="calendar">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(252, 233, 79);

</string>
            </property>
            <property name="minimumDate">
             <date>
              <year>2000</year>
              <month>9</month>
              <day>14</day>
             </date>
            </property>
            <property name="gridVisible">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget">
            <property name="styleSheet">
             <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(252, 233, 79, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(164, 0, 0);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="activityPage">
       <property name="styleSheet">
        <string notr="true">background-image: url(:/pozadina/slikeCalendar/14.jpg);</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Plan your day</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgba(251, 215, 161, 102);</string>
                    </property>
                    <property name="text">
                     <string>6h - 9h:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgba(251, 215, 161, 102);</string>
                    </property>
                    <property name="text">
                     <string>9h - 12h:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_4">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgba(251, 215, 161, 102);</string>
                    </property>
                    <property name="text">
                     <string>12h - 15h:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_5">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgba(251, 215, 161, 102);</string>
                    </property>
                    <property name="text">
                     <string>15h - 18h:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_6">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgba(251, 215, 161, 102);</string>
                    </property>
                    <property name="text">
                     <string>18h - 21h:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_7">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgba(251, 215, 161, 102);</string>
                    </property>
                    <property name="text">
                     <string>21h - 24h:</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <widget class="QPlainTextEdit" name="text6_9">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgba(251, 215, 161, 102);</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPlainTextEdit" name="text9_12">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgba(251, 215, 161, 102);</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPlainTextEdit" name="text12_15">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgba(251, 215, 161, 102);</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPlainTextEdit" name="text15_18">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgba(251, 215, 161, 102);</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPlainTextEdit" name="text18_21">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgba(251, 215, 161, 102);</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPlainTextEdit" name="text21_24">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgba(251, 215, 161, 102);</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QCheckBox" name="alarm_chech_box">
                <property name="styleSheet">
                 <string notr="true">background-image: url(:/pozadina/slikeCalendar/14.jpg);</string>
                </property>
                <property name="text">
                 <string>Set Reminder</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTimeEdit" name="alarm_time">
                <property name="styleSheet">
                 <string notr="true">background-image: url(:/pozadina/slikeCalendar/9.jpg);</string>
                </property>
                <property name="displayFormat">
                 <string>HH:mm:ss</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QDialogButtonBox" name="buttonBox">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(203, 224, 255);</string>
            </property>
            <property name="standardButtons">
             <set>QDialogButtonBox::Cancel|QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>430</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew_Activities">
    <property name="title">
     <string>New Activities</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionEdit"/>
    <addaction name="actionRemove"/>
   </widget>
   <addaction name="menuNew_Activities"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
